modules/consul-cluster/variables.tf

```diff
@@ -184,3 +184,9 @@ variable "ssh_port" {
   description = "The port used for SSH connections"
   default     = 22
 }
+
+variable "tags" {
+  description = "List fo extra tag blocks added to the autoscaling group configuration"
+  type        = "list"
+  default     = []
```

Comment:
Could you add an example? E.g.

```hcl
  default = []
  # Example:
  #
  # default = [{ ... } ]
```

Reply:
I did add an example in the `main.tf` (in the root of the repository). Where else would you like to have it in?